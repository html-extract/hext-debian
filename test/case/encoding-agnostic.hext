<p @text*="é‚£" @text:content />
