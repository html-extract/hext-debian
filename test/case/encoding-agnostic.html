<html>
  <head>
  </head>
  <body>
    <p>
      爱因斯坦，扮成罗宾汉<br>
      随着他在树干的回忆<br>
      一个小时前就这样过去了<br>
      随着他的朋友，一个嫉妒的和尚<br>
      他看上去是那么一尘不染可怕<br>
      当他早起床实在香烟<br>
      然后，他去了嗅探排水管<br>
      而背诵字母表<br>
      现在，你不会想看看他<br>
      但他成名已久以前<br>
      对于玩电小提琴<br>
      在荒凉行<br>
    </p>
    <p>
      爱因斯坦，扮成罗宾汉<br>
      随着他在树干的回忆<br>
      一个小时前就这样过去了<br>
      随着他的朋友，一个嫉妒的和尚<br>
      他看上去是 么一尘不染可怕<br>
      当他早起床实在香烟<br>
      然后，他去了嗅探排水管<br>
      而背诵字母表<br>
      现在，你不会想看看他<br>
      但他成名已久以前<br>
      对于玩电小提琴<br>
      在荒凉行<br>
    </p>
  </body>
</html>
