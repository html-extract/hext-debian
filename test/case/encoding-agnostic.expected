{"content":"爱因斯坦，扮成罗宾汉 随着他在树干的回忆 一个小时前就这样过去了 随着他的朋友，一个嫉妒的和尚 他看上去是那么一尘不染可怕 当他早起床实在香烟 然后，他去了嗅探排水管 而背诵字母表 现在，你不会想看看他 但他成名已久以前 对于玩电小提琴 在荒凉行"}
